src/react-apps/applications/ux-editor/src/components/index.ts

```diff
@@ -15,8 +15,23 @@ export interface IComponent {
   Tag: any;
   Type: ComponentTypes;
   customProperties?: any;
+  Icon: string;
 }
 
+// export interface IComponentIcons {
+//   Header: string;
+//   Paragraph: string;
+//   Input: string;
+//   Datepicker: string;
+//   Dropdown: string;
+//   Checkboxes: string;
+//   RadioButtons: string;
+//   TextArea: string;
+//   FileUpload: string;
+//   Submit: string;
+//   AddressComponent: string;
+// }
```

Comment:
Why is this commented out?

Reply:
Was not sure wheter we needed an interface or if we should just set the type to any. Removed the commented out code